package.json

```diff
@@ -8,7 +8,7 @@
     "build": "./script/build.sh",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
     "build:compare": "node script/content-build-compare.js && diff --color websiteContentBuildHash.txt standaloneContentBuildHash.txt",
-    "build:content": "node --max-old-space-size=4096 script/build-content.js",
+    "build:content": "node --max-old-space-size=8192 script/build-content.js",
```

Comment:
FYSA - it's the `parse-html` build step that uses a ton of memory during the build (it parses the file contents of every page into a DOM using JS-DOM, iirc)

Reply:
So the parsed DOM of hundred of pages are in memory all at once....definitely some room to optimize this